client/my-sites/email/email-providers-stacked-comparison/style.scss

```diff
@@ -30,3 +33,10 @@
 	}
 }
 
+.email-providers-stacked-comparison__main {
+	padding: 71px 20px 20px;
+	@include break-mobile {
+		padding: 71px 24px 24px;
+	}
```

Comment:
Question: where is the 71px padding coming from?

This number feels _extremely_ magical and arbitrary, so I want to understand where it is coming from and why we are using it.